model.js

```diff
@@ -21,4 +21,14 @@ exports.selectReviewById = (review_id) => {
     .then((result) => {
         return (result.rows)
     })
+}
+
+exports.selectCommentById = (review_id) => {
```

Comment:
This query is vunerable to SQL injection using a string literal from a user input, how could we change this to protect yourself against a malicious input eg. '; drop tables'